lib/developer_portal/lib/liquid/filters/rails_helpers.rb

```diff
@@ -238,7 +238,7 @@ def self.replace_googleapis(url)
         return url unless Rails.env.test?
 
         if url =~ %r{//ajax.googleapis.com/ajax/libs/jquery/(.+?)/jquery.min.js}
-          "vendor/jquery-#{$1}.min.js"
+          "essential_assets/jquery-#{$1}.min.js"
```

Comment:
OMG, why did we have googleapis in the first place??!

Looks like there is still some leftover: https://github.com/3scale/porta/blob/36eeb457a682b3a1e96d99cdf91738551bf4482c/app/views/stats/days/show.html.erb#L1

I think in the next PR we'd need to remove both this helper, and that external JS.